src/System.ServiceModel.Security/tests/ServiceModel/SecutityBindingElementTest.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
You can remove one of these licensing comments.

Reply:
Thanks! Good catch! I must have copied and accidently paste twice. Updated now.